Sync/Synchronizers/ClientsSynchronizer.swift

```diff
@@ -83,20 +83,49 @@ open class DisplayURICommand: Command {
     }
 }
 
+open class RepairResponseCommand: Command {
+    let repairResponse: RepairResponse
+
+    public init?(command: String, args: [JSON]) {
```

Comment:
Nit: Doesn't need to have a failable initalizer unless we want`RepairResponse.fromJSON` to throw something.